manifest-schema.graphql

```diff
@@ -42,6 +42,7 @@ type EthereumContractAbi {
 
 type EthereumBlockHandler {
   handler: String!
+  input: String
```

Comment:
What is the `input` expected to be? Also, can we add a validation test under `tests/cli/validation/`?

Reply:
The variants for input are `Block`, `FullBlock` and `FullBlockWithReceipts` with it defaulting to `Block` if nothing is specified. 

Sure, I'll add a validation test. :)

Reply:
I don't like these names, they are not consistent with the spelling of other manifest fields. And they sound too coupled with the actual data types.

So, I'd prefer something like
```yaml
blockFormat: block-only
blockFormat: block-with-transactions
blockFormat: block-with-receipts
```
Perhaps @Zerim can weigh in from a product / experience perspective.

Reply:
In general, I agree with @Jannis here. We don't use PascalCase anywhere else in the manifest for values that are defined by the protocol. I.e., we use things like `ethereum/contract` or `ethereum/events`.

I also think `input` is overly broad and that `blockFormat` or alternatively have some sort of `handler>type: block-with-receipts` or `handler>type:block-with-receipts`.... but we seem to have gone with the pattern of giving each handler type their own *key* in the manfiest rather than being a value.

Reply:
I'm open to different names. :)
Now that you mention it, I agree that it's better to replace FullBlock with Block. However, I'm not following the suggestion to use hyphens. Where else in the manifest are they used?  
Also, does the field really control formatting? How about `blockData` or `blockType`?

Reply:
I see that in the network field of the manifest values use a hyphen like: mainnet, poa-core, poa-sokol. Not sure if present in other places.